### PR TITLE
Fusing live data

### PR DESCRIPTION
**Pull request by Debby486**

### CODE REVIEW

File: src/screens/HomeScreen.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  const goToDetails = () => {
+    navigation.navigate('Details');
+  }
+  useEffect(() => {
+    const getApi = async () => {
+      setCurrencies(await getRequest());
+    }
+    getApi();
+  }, [setCurrencies])
+console.log('>>>>>', currencies)
```
Comment (rebirthtobi) [inline src/screens/HomeScreen.js:75]: remove console.log

File: src/screens/HomeScreen.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import React, {useState, useEffect} from 'react';
+import { StyleSheet, View, Button, Text, ScrollView } from 'react-native';
+import { Card} from 'react-native-elements'
+import CardComponent from '../components/Card';
+import getRequest from '../../lib/request';
+
+const currencies = [
```
Comment (rebirthtobi) [inline src/screens/HomeScreen.js:7]: remove dummy data

File: src/screens/HomeScreen.js (hunk 1 above)
Comment (Debby486) [reply src/screens/HomeScreen.js]: I've fixed it

File: src/screens/HomeScreen.js (hunk 2 above)
Comment (Debby486) [reply src/screens/HomeScreen.js]: I've fixed this too

File: src/screens/HomeScreen.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    const getApi = async () => {
+      setCurrencies(await getRequest());
+    }
+    getApi();
+  }, [setCurrencies])
+    
+  return (
+    <View style={styles.container}>
+      <ScrollView>
+    <Card containerStyle={styles.card} title='Total Value' titleStyle={{marginTop: 20, color: '#fff', fontWeight: 'normal', fontSize: 14}} dividerStyle={{backgroundColor: 'pink' }}>
```
Comment (rebirthtobi) [inline src/screens/HomeScreen.js:23]: Inline styles is bad and should be avoided at all cost

**Pull request merged**